README.md

```diff
@@ -83,16 +83,33 @@ func main() {
 		)
 	}
 
-	err := gormbulk.BulkInsert(db, insertRecords, 3000)
+	err = gormbulk.BulkInsert(db, insertRecords, 3000)
 	if err != nil {
 		// do something
 	}
 
 	// columns you want to exclude from Insert, specify as an argument
 	err = gormbulk.BulkInsert(db, insertRecords, 3000, "Email")
-        if err != nil {
-            // do something
-        }
+	if err != nil {
+		// do something
+	}
+
+	// get inserted record id
+	var ids []uint
+	afterInsert := func(db *gorm.DB) error {
```

Comment:
I appreciate your cooperation to update README, but can I have a chance to change README as a follow-up task after merging? I actually want to review entire sentences.